torch/distributed/_tensor/random.py

```diff
@@ -11,64 +11,67 @@
 from torch.distributed._tensor.placement_types import DTensorSpec, Shard
 
 
-def set_rng_state(new_state: Tensor, device_mesh: DeviceMesh) -> None:
+def set_rng_state(device_mesh: DeviceMesh, seed: int, offset: int) -> None:
```

Comment:
I think this file exposes too many methods and we should probably only have one class (i.e. RNGStateTracker) that encapsulates all the seed/offset calculation logic,